plugins/modules/yarn.py

```diff
@@ -79,88 +78,61 @@
   state:
     type: str
     description:
-      - Installation state of the named node.js library
-      - If absent is selected, a name option must be provided
+      - Installation state of the named Node.js library.
+      - If V(absent) is selected, a O(name) option must be provided.
     required: false
     default: present
-    choices: [ "present", "absent", "latest" ]
+    choices: ["present", "absent", "latest"]
 requirements:
-    - Yarn installed in bin path (typically /usr/local/bin)
-'''
-
-EXAMPLES = '''
-- name: Install "imagemin" node.js package.
-  community.general.yarn:
-    name: imagemin
-    path: /app/location
-
-- name: Install "imagemin" node.js package on version 5.3.1
-  community.general.yarn:
-    name: imagemin
-    version: '5.3.1'
-    path: /app/location
-
-- name: Install "imagemin" node.js package globally.
-  community.general.yarn:
-    name: imagemin
-    global: true
-
-- name: Remove the globally-installed package "imagemin".
-  community.general.yarn:
-    name: imagemin
-    global: true
-    state: absent
-
-- name: Install "imagemin" node.js package from custom registry.
-  community.general.yarn:
-    name: imagemin
-    registry: 'http://registry.mysite.com'
-
-- name: Install packages based on package.json.
-  community.general.yarn:
-    path: /app/location
-
-- name: Update all packages in package.json to their latest version.
-  community.general.yarn:
-    path: /app/location
-    state: latest
-'''
-
-RETURN = '''
-changed:
-    description: Whether Yarn changed any package data
-    returned: always
-    type: bool
-    sample: true
-msg:
-    description: Provides an error message if Yarn syntax was incorrect
-    returned: failure
-    type: str
-    sample: "Package must be explicitly named when uninstalling."
-invocation:
-    description: Parameters and values used during execution
-    returned: success
-    type: dict
-    sample: {
-            "module_args": {
-                "executable": null,
-                "globally": false,
-                "ignore_scripts": false,
-                "name": null,
-                "path": "/some/path/folder",
-                "production": false,
-                "registry": null,
-                "state": "present",
-                "version": null
-            }
-        }
+  - Yarn installed in bin path (typically C(/usr/local/bin))
+"""
+
+EXAMPLES = r"""
+  - name: Install "imagemin" Node.js package.
```

Comment:
This now has extra indentation that shouldn't be there. Can you please remove the extra indent in the examples? Thanks.

Reply:
Yeah, this was done automatically. I am using ruamel.yaml and it has no provision for lists being the top levelof the document. But it is doable so, coming soon.